input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,395 @@
-                    Immunization.patient.reference
-                </p>
-            </td>
-            <td width="78" valign="top">
-                <p>
-                    <a
-                        href="http://hl7.org/fhir/us/core/StructureDefinition/us-core-immunization"
-                    >
-                        US Core Immunization
-                    </a>
-                </p>
-            </td>
-            <td width="137" valign="top">
-                <p>
-                    <img src="https://hl7.org/fhir/R4/assets/images/tick.png"/> noting that US Core extensibly binds to CVX which is
-                    not an AU vaccine terminology
-                </p>
-            </td>
+            <td style="width: 25%; text-align: center; vertical-align: middle;"><a href="StructureDefinition-au-core-smokingstatus.html">AU Core Smoking Status</a></td>
+            <td style="width: 25%; text-align: center; vertical-align: middle;"><img src="green_checkmark.svg.png" width="20"/></td>
+            <td style="width: 25%; text-align: center; vertical-align: middle;"><img src="cross_red_circle.svg.png" width="20"/></td>
+            <td style="width: 25%; text-align: center; vertical-align: middle;"><img src="green_checkmark.svg.png" width="20"/></td>
         </tr>
         <tr>
-            <td width="93" valign="top">
-                <p>
-                    <a
-                        href="StructureDefinition-au-core-location.html"
-                    >
-                        AU Core Location
-                    </a>
-                </p>
-            </td>
-            <td width="81" valign="top">
-            </td>
-            <td width="111" valign="top">
-            </td>
-            <td width="68" valign="top">
-            </td>
-            <td width="130" valign="top">
-            </td>
-            <td width="78" valign="top">
-                <p>
-                    <a
-                        href="http://hl7.org/fhir/us/core/StructureDefinition/us-core-location"
-                    >
-                        US Core Location
-                    </a>
-                </p>
-            </td>
-            <td width="137" valign="top">
-                <p>
-                    <img src="https://hl7.org/fhir/R4/assets/images/tick.png"/> if Location.name is supplied
-                </p>
-            </td>
+            <td style="width: 25%; text-align: center; vertical-align: middle;"><a href="StructureDefinition-au-core-waistcircum.html">AU Core Waist Circumference</a></td>
+            <td style="width: 25%; text-align: center; vertical-align: middle;"><img src="green_checkmark.svg.png" width="20"/></td>
+            <td style="width: 25%; text-align: center; vertical-align: middle;">&nbsp;</td>
+            <td style="width: 25%; text-align: center; vertical-align: middle;"><img src="green_checkmark.svg.png" width="20"/></td>
         </tr>
+    </tbody>
+</table>
+
+#### International Patient Access 1.0.0
```

Comment:
1. Remove the version from the heading.
2. Have a sentence on IPA with the version, see relationships page with summary table on the IG for a bit of inspiration.
3. Have an intro sentence on what this section does.

Reply:
Done - and looking forward to the feedback.

Reply:
1. Confirmed
2. Confirmed
3. Replace "AU Core is designed to be compatible with IPA, e.g. AU Core compliant data can be accessed by an IPA conformant client. To achieve compliance with both AU Core and IPA, additional requirements may need to be met. These requirements are outlined below."

With "The following IPA profile(s) contain incompatible requirements with the equivalent AU Core profile. Implementers are advised to note that substantial code changes may be required to support these profiles."

5. Missing Data & Supressed Data are profile compliance requirements, and subject to same determination, i.e.: compliant, additional requirements, incompatible.

When presenting this content make it very short and clear, as in, if an IG doesn't have one or both of these, a short note that the IG does not contain any requirements on Missing Data / Suppressed Data can just be stated.

Where a spec has one or both, then given that some profiles have green ticks you must have determined that when compliant to AU Core you are compliant to the other spec. Thus add a simple statement to that effect using wording from the legend.
6. Confirmed

Reply:
Done